usaspending_api/etl/transaction_loaders/data_load_helpers.py

```diff
@@ -11,9 +11,11 @@
 
 
 def capitalize_if_string(val):
```

Comment:
Name is misleading since it does more than capitalize a value if it is a string.

Reply:
In retrospect that was a mistake. Function has been changed back to original purpose.